src/ErrorHandler.php

```diff
@@ -451,7 +453,9 @@ private function handleException(\Throwable $exception): void
             throw $exception;
         }
 
-        $this->handleException($previousExceptionHandlerException);
+        if ($exception !== $previousExceptionHandlerException) {
+            $this->handleException($previousExceptionHandlerException);
+        }
```

Comment:
This is another helpful change: let's not go into recursion if we have the same exception we were reporting on.

Reply:
It's not needed because of the `if` statement right above these lines of code that throws if the exception thrown by the previous handler is the same as the one we're trying to handle. To say it simple, this case was already covered